libmanager/libManager.go

```diff
@@ -27,7 +45,54 @@ func FillLibrary(r *models.ScanResult) (totalCnt int, err error) {
 		}
 		totalCnt += len(vinfos)
 	}
-	db.Close()
 
 	return totalCnt, nil
 }
+
+func downloadDB(appVersion, cacheDir string, quiet, light, skipUpdate bool) error {
+	client := initializeDBClient(cacheDir, quiet)
+	ctx := context.Background()
+	needsUpdate, err := client.NeedsUpdate(appVersion, light, skipUpdate)
+	if err != nil {
+		return xerrors.Errorf("database error: %w", err)
+	}
+
+	if needsUpdate {
+		log.Logger.Info("Need to update DB")
```

Comment:
A `quiet` flag looks suppress any logs.
I expected `-suppress-progress` or so if it only suppresses a progress bar.

Reply:
Thanks, add --no-progress flag to tui/report cmdl.